### PR TITLE
Update: Automatic payments type

### PR DESCRIPTION
**Pull request by maxplt**
# Jira Issue: []

## What?
Disable debit status checking on non-Sweden claims

## Why?
-

## Optional checklist
- [ ] Updated `src/changelog.ts`
- [ ] Codescouted
- [ ] Unit tests written
- [ ] Tested locally

### CODE REVIEW

File: src/portals/hope/features/claims/claim-details/SubclaimPayments/components/SubclaimPaymentForm.tsx [hunk 1]
```diff
@@ -172,8 +173,10 @@ export const SubclaimPaymentForm: React.FC<{
   const member = data?.claim?.member
 
   const isPaymentActivated =
-    !!member?.directDebitStatus?.activated ||
-    !!member?.payoutMethodStatus?.activated
+    member?.contractMarketInfo?.market === Market.Norway || Market.Denmark
```
[the remark this answers is not in the document]
Comment (maxplt) [reply src/portals/hope/features/claims/claim-details/SubclaimPayments/components/SubclaimPaymentForm.tsx:177]: 😅

File: src/portals/hope/features/claims/claim-details/SubclaimPayments/components/SubclaimPaymentForm.tsx [hunk 2]
```diff
@@ -172,8 +173,10 @@ export const SubclaimPaymentForm: React.FC<{
...
+      ? true
+      : !!member?.directDebitStatus?.activated ||
+        !!member?.payoutMethodStatus?.activated
```
Comment (falklinus) [inline src/portals/hope/features/claims/claim-details/SubclaimPayments/components/SubclaimPaymentForm.tsx:180]:
```suggestion
    member?.contractMarketInfo?.market === Market.Sweden
      ? !!member?.directDebitStatus?.activated ||
        !!member?.payoutMethodStatus?.activated
      : false
```

**Review by falklinus — COMMENTED** [review]
I think the turnary operation is a bit off 🤔

File: src/portals/hope/features/claims/claim-details/SubclaimPayments/components/SubclaimPaymentForm.tsx (hunk 2 above)
Comment (falklinus) [reply src/portals/hope/features/claims/claim-details/SubclaimPayments/components/SubclaimPaymentForm.tsx:180]: I was a bit slow, didn't see the above comments 😅

**Review by vonElfvin — APPROVED** [review]

**Pull request merged**